control-plane/subcommand/install-cni/binary.go

```diff
@@ -0,0 +1,56 @@
+package installcni
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// copyFile copies a file from a source directory to a destination directory.
+func copyFile(srcFile, destDir string) error {
+	// If the src file does not exist then either the incorrect command line argument was used or
+	// the docker container we built is broken somehow.
+	if _, err := os.Stat(srcFile); os.IsNotExist(err) {
+		return err
+	}
```

Comment:
Instead of having this info in a comment, I think it would be valuable to append it to the existing error so that it's useful for debugging. 

```suggestion
	// If the src file does not exist then either the incorrect command line argument was used or
	// the docker container we built is broken somehow.
	if _, err := os.Stat(srcFile); os.IsNotExist(err) {
		return fmt.Errorf("%v. Check that the correct command line argument was used and verify that the container image does not contain the source file.", err)
	}
```